packages/rum-core/src/rumEvent.types.ts

```diff
@@ -131,6 +131,10 @@ export type RumErrorEvent = CommonProperties & {
      * The type of the error
      */
     readonly type?: string
+    /**
+     * Whether the error has been handled manually in the source code or not
+     */
+    readonly handling?: 'handled' | 'unhandled'
```

Comment:
should we also add this field to logs error?

Reply:
I've asked Maxime and for him, it's good enough for now.